spec/spec.md

```diff
@@ -1176,6 +1175,28 @@ When using this [[ref:Feature]]:
   property. If present, its value ****MUST**** be a
   [JSON LD Framing Document](https://w3c.github.io/json-ld-framing/) object.
 
+### Retention Feature
+
+``The Retention [[ref:Feature]] extends the [[ref:Input Descriptor Object]]'s _field_ 
+object, allowing a Verifier to indicate it will retain the submitted value for the
+specific field.  It is currently presented to support mDL systems, and may be deprecated in the
+ future or a separate specification for more robust mDL interoperability is created at a later time.
```

Comment:
I'm having trouble parsing the sentence that begins "It is currently . . ."  I think the 'or' maybe should be an 'if'?